app/Http/Controllers/ActorController.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\Actor\CreateRequest;
+use App\Http\Requests\Actor\EditRequest;
+use App\Models\Actor;
+use Illuminate\Http\Request;
+
+class ActorController extends Controller
+{
+    public function createForm()
+    {
+        return view('actors.create');
+    }
+
+    public function editForm(int $id)
+    {
+        $actor = Actor::query()->findOrFail($id);
```

Comment:
может лучше сделать через роут биндинг?
```suggestion
    public function editForm(Actor $actor)
    {
```

Reply:
и в остальных методах тоже